package/MDAnalysis/coordinates/XYZ.py

```diff
@@ -367,8 +367,10 @@ def _read_next_timestep(self, ts=None):
             # we assume that there are only two header lines per frame
             f.readline()
             f.readline()
+            tmp_buf = []
```

Comment:
one comment line referring to speedup and perhaps "See PDB.py" or refer to GH discussion.